.travis.yml

```diff
@@ -12,33 +12,32 @@ sudo: false
 
 cache:
   directories:
-    - $HOME/.composer/cache
+    - $HOME/.composer/cache/files
  
 env:
-  - SYMFONY_VERSION=2.7.* SYMFONY_DEPRECATIONS_HELPER=weak
+  global:
+    - SYMFONY_DEPRECATIONS_HELPER=309
 
 matrix:
   include:
     - php: 5.3
-      env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
+      env: COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.6
-      env: SYMFONY_VERSION=2.3.* SYMFONY_DEPRECATIONS_HELPER=weak
+      env: SYMFONY_VERSION=2.3.*
     - php: 5.6
-      env: SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=309
+      env: SYMFONY_VERSION=2.7.*
     - php: 5.6
-      env: SYMFONY_VERSION=3.0.*
-  allow_failures:
-    - env: SYMFONY_VERSION=3.0.*
+      env: DEPS=dev
```

Comment:
all builds now run with stable versions. This will (a) speed up composer a lot and (b) simulate a normal install of the bundle in an application. Only one build will install the dev dependencies.